training/src/anemoi/training/config/model/graphtransformer_diffusion.yaml

```diff
@@ -0,0 +1,107 @@
+num_channels: 1024
+cpu_offload: False
+
+keep_batch_sharded: True
+
+model:
+  _target_: anemoi.models.models.AnemoiDiffusionModelEncProcDec
+  # Diffusion parameters
+  diffusion:
+    sigma_data: 1.0
+    noise_channels: 32
+    noise_cond_dim: 16
+    sigma_max: 100.0
+    sigma_min: 0.02
+    rho: 7.0
+    noise_embedder:
```

Comment:
I think this new interface for the config looks much better and addresses the points raised by @icedoom888 , thanks @ssmmnn11 !